graph.js

```diff
@@ -410,6 +441,19 @@ function noResults(neoData) {
   return false;
 }
 
+function extractUserLocation(row) {
+  const [location, country, city, postal] = row;
+
+  return Object.assign(
+    {},
```

Comment:
If we wanted to be explicit about what we are getting out of location we could do something like this (also we do not need `Object.assign` in this case):
```
return (
  { id: location.id,
    name: location.name,
    country: country.name,
    city: city.name,
    postal: postal.name }
)
```